.github/dependabot.yml

```diff
@@ -707,3 +707,9 @@ updates:
     directory: /docker/xpanse-ml-ev2
     schedule:
      interval: daily
+  - package-ecosystem: pip
+    directory: /docker/stringsifter
+    schedule:
+      interval: daily
+    ignore:
+      - dependency-name: "lightgbm"
```

Comment:
why ignore?

Reply:
@wissamg did it, not sure why, what exactly does it do?

Reply:
will not allow dependabot to update this subdependency